nbgrader/docs/source/command_line_tools/index.rst

```diff
@@ -21,7 +21,7 @@ Instructor commands
     nbgrader-validate
     nbgrader-update
     nbgrader-autograde
-    nbgrader-feedback
+    nbgrader-generate-feedback
```

Comment:
Is this correct? It doesn't look like you've renamed the command line app to `nbgrader generate_feedback` as far as I can tell---isn't it still `nbgrader feedback`?

Reply:
Done